### DISCUSSION TITLE
Adding a stiemap

### DISCUSSION THREAD

**Discussion by rosiepy** [requarks/wiki#7099] [General]
Hello,
I was wondering if I already have the urls in sitemap format, how can I add it to wiki.js so that it's not displayed to the user but accessible through domain.com/sitemap.xml?

**Comment by aapwikip** [comment]
Upload the file 'sitemap.xml' as an asset like images

**Reply by rosiepy** [reply]
In order to upload an asset like an image I have to do it on markdown right? There it provides the link but does not actually create a url domain.com/sitemap.xml with the contents of the file.

**Comment by Philipinho** [comment]
You can try the wiki.js sitemap generator I wrote: https://github.com/hostwiki/wikijs-sitemap.

**Reply by rosiepy** [reply]
Does is work with AWS?

**Reply by Philipinho** [reply]
> Does is work with AWS?

Yes, it should work on any environment you can run Nodejs or docker.

**Comment by eethersway** [comment]
@Philipinho 

Excuse the non dev request, but is there any way that you could assist in creating a sitemap for https://wiki.influenceth.io/

I have been working with chatGPT and CursorAI to try and get everything to work, but alas I have yet to get it to work.

**Reply by Philipinho** [reply]
Have you tried the Wiki.js sitemap generator I wrote? https://github.com/hostwiki/wikijs-sitemap
The readme contains full setup instructions.
Let me know If you encounter any issues.
